init/onboard-msisdn-oracle/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine
+
+WORKDIR /src/
+
+RUN apk add curl jq
+
+COPY ./insert_MSISDNS /src/
+
+RUN chmod +x /src/insert_MSISDNS
+
+CMD . ./insert_MSISDNS
```

Comment:
I would group up the middle 4 maybe. @msk- ?

Reply:
What do you mean?